hieradata/common.yaml

```diff
@@ -1 +1,19 @@
-classes: []
+classes:
+    - ocf::apt
+    - ocf::auth
+    - ocf::autologout
+    - ocf::groups
+    - ocf::kerberos
+    - ocf::ldap
+    - ocf::locale
+    - ocf::motd
+    - ocf::munin::node
+    - ocf::networking
+    - ocf::packages
+    - ocf::puppet
+    - ocf::rootpw
+    - ocf::staff_users
+    - ocf::systemd
+    - ocf::utils
+
+packages: {}
```

Comment:
I'd vote for not having it at all then, seems like a rabbit hole I really don't want to go down.